card.io/src/main/jni/Android.mk

```diff
@@ -11,16 +11,16 @@ LOCAL_PATH := $(call my-dir)
 LOCAL_DMZ_DIR := card.io-dmz
 
 # --- declare opencv prebuilt static libs ---------------------------------
-ifneq (,$(filter $(TARGET_ARCH_ABI),armeabi-v7a))
+ifneq (,$(filter $(TARGET_ARCH_ABI),armeabi-v7a x86))
```

Comment:
I don't think so. The last argument is supposed to be a plain string. filter will look for the presence of the current arch into that string and return armeabi-v7a or x86 if it's the current arch. In these cases the return value will not be empty and the block will be executed, as expected.